### PR TITLE
lvcreate, lvextend, lvreduce, lvremove, lvresize, lvs, pvcreate, pvs, vgcreate, vgs: harmonize descriptions

### PR DESCRIPTION
**Pull request by aherst**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Harmonized descriptions by:
- indicating it is one of the LVM tools so it will be returned by "tldr -s lvm"
- adding or changing the More information link to point to a guide in the RHEL documentation

### CODE REVIEW

**Comment by bl-ue** [comment]
@ahert these RHEL 8 links are all 404s for me.  :thinkin: You could change them to RHEL 7...

**Comment by bl-ue** [comment]
Hmm...even with `/index` it doesn't work.

**Comment by aherst** [comment]
I need to replace the complete URL. They changed more than the version number in it. Serves me right for not checking first.

**Comment by bl-ue** [comment]
Yeah, it would be nice to use the latest version! 👍🏻

**Comment by aherst** [comment]
Got them all. The URL will resolve without the trailing index. Should I remove it to reduce the size of the URL?

**Comment by bl-ue** [comment]
Sure. I would.

**Comment by aherst** [comment]
I think these are good to go now.

File: pages/linux/lvcreate.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # lvcreate
 
-> Creates a logical volume in an existing volume group.
-> A volume group is a collection of logical and physical volumes.
+> Creates a logical volume in an existing volume group. A volume group is a collection of logical and physical volumes.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/lvcreate.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/lvextend.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 > Increase the size of a logical volume.
 > One of the Logical Volume Manager (LVM) tools.
-> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration>.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/lvextend.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/lvreduce.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # lvreduce
 
 > Reduce the size of a logical volume.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/lvreduce.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/lvremove.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
 # lvremove
 
 > Remove one or more logical volumes.
-> More information: <https://sourceware.org/lvm2/>.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/lvremove.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/lvresize.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
 # lvresize
 
 > Change the size of a logical volume.
-> More information: <https://man7.org/linux/man-pages/man8/lvresize.8.html>.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/lvresize.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/lvs.md [hunk 6]
```diff
@@ -1,7 +1,8 @@
 # lvs
 
-> Display information about LVM logical volumes.
-> More information: <https://www.man7.org/linux/man-pages/man8/lvs.8.html>.
+> Display information about logical volumes.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/lvs.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/pvcreate.md [hunk 7]
```diff
@@ -1,6 +1,8 @@
 # pvcreate
 
-> Initialize a physical volume (disk or partition) for use by the Logical Volume Manager (LVM).
+> Initialize a disk or partition for use as a physical volume.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/pvcreate.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/pvs.md [hunk 8]
```diff
@@ -1,7 +1,8 @@
 # pvs
 
-> Display information about LVM physical volumes.
-> More information: <https://www.man7.org/linux/man-pages/man8/pvs.8.html>.
+> Display information about physical volumes.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/pvs.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/vgcreate.md [hunk 9]
```diff
@@ -1,6 +1,8 @@
 # vgcreate
 
 > Create volume groups combining multiple mass-storage devices.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/vgcreate.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

File: pages/linux/vgs.md [hunk 10]
```diff
@@ -1,7 +1,8 @@
 # vgs
 
-> Display information about LVM volume groups.
-> More information: <https://man7.org/linux/man-pages/man8/vgs.8.html>.
+> Display information about volume groups.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
```
Comment (bl-ue) [inline pages/linux/vgs.md:5]:
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes>.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
The URL could be even shorter if you remove the trailing slash.  I also like the look of it better.

**Comment by navarroaxel** [comment]
> The URL could be even short if you remove the trailing slash. I also like the look of it better.

Wait @bl-ue, I think the URLs should be friendly with `curl`, without extra arguments, so the URL shouldn't have redirections.

**Comment by bl-ue** [comment]
But we don't do that in the other pages...  Usually short URLs redirect to longer ones that have components based on the browser, etc.  For example, language.  Many URLs have a locale in them but it can be omitted, and is redirected to a URL with the locale in.  We do that a lot here.

**Comment by navarroaxel** [comment]
That's right, I think the auto i18n worth more than a clean URL.

**Comment by bl-ue** [comment]
So should we still keep the trailing '/'?

File: pages/linux/lvresize.md [hunk 11]
```diff
@@ -1,7 +1,8 @@
 # lvresize
 
 > Change the size of a logical volume.
-> More information: <https://man7.org/linux/man-pages/man8/lvresize.8.html>.
```
Comment (sbrl) [inline pages/linux/lvresize.md:4]: I'm really reluctant to change this URL from man7.org - a neutral site - to redhat.com - a distribution (and distribution version specific) URL. I wasn't aware that man7.org pages were a thing for LVM tools, but now that I am it seems like a much better alternative than the redhat docs.

Comment (aherst) [reply pages/linux/lvresize.md:4]: The reason I prefer the RHEL URL is that it provides a conceptual guide. I can always use man if I want the man page.

Comment (bl-ue) [reply pages/linux/lvresize.md:4]: Yeah, @sbrl it's not a link to a man page; it's a link to a guide.  I'm kindof in with @aherst.

Comment (bl-ue) [reply pages/linux/lvresize.md:4]: It doesn't really seem that useful to link to a man page when a good, formal (and relatively official) tutorial exists.  The manpage is good if there's nothing else, but...those are my thoughts.

Comment (aherst) [reply pages/linux/lvresize.md:4]: Here's another option. For each of the lvm tools I will set the More information links point to https://man7.org/linux/man-pages/man8/lvm.8.html. That will provide contextualization for each of the tools. When I write the lvm tldr-page I'll set its More information link to the RHEL guide. Comments?

Comment (bl-ue) [reply pages/linux/lvresize.md:4]: Maybe you could keep each respective manpage, but add
```
See also: `lvm`
```
Above the More information line.  I like that the best&mdash;what do you think?

Comment (aherst) [reply pages/linux/lvresize.md:4]: OK. "See also: `lvm`" is much more concise than my proposal. 

I'll change the More Information lines to the man page on man7.org. I do still think pointing to a non-man page will add more value. I'll chime in if and when the subject comes up for discussion again. 

I did learn a valuable lesson about not confounding two issues in one PR :-). My original intent was just to make the results returned from "tldr -s lvm" to be exhaustive. Adding and harmonizing the More info lines was an after thought. I'll keep this in mind moving forward.

**Review by sbrl — APPROVED** [review]
Thanks, @aherst!

@bl-ue, are we ok to merge here?

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**